qa/tasks/deepsea.py

```diff
@@ -68,6 +76,7 @@ def _check_config_key(key, default_value):
             if key not in self.config or not self.config[key]:
                 self.config[key] = default_value
 
+        _check_config_key('install', 'source')
```

Comment:
I thought you wanted to default to `package` ??

Reply:
I served your desires :-)